spec/BehavioralFsm.spec.js

```diff
@@ -1067,6 +1067,30 @@ function runBehavioralFsmSpec( description, fsmFactory ) {
 						}
 					] );
 				} );
+				( Promise ? it : it.skip )( "should return a promise when an action is deferred", function( done ) {
```

Comment:
Unsure how you'd like this handled. I figured that for the moment I'd be safer, and just skip the test if Promise isn't available but this might not be the best option.